benchmarks/accuracy/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+enable_testing()
+
+find_package(GTest CONFIG REQUIRED)
+find_package(TBB CONFIG REQUIRED)
```

Comment:
Are you sure you want to _force_ the use of [a config file](https://cmake.org/cmake/help/v3.16/command/find_package.html)? :thinking: The comment applies to most `find_package` that  remains in the review.